projects/mysadna/src/app/project-details/project-details.component.scss

```diff
@@ -0,0 +1,51 @@
+.project-details {
+  text-align: center;
+  >.project-name {
+    margin: 1.5em;
+  }
+  >.project-description {
+    font-weight: normal;
+    color: #4a4a4a;
+  }
+  >.project-link {
+    h3 {
+      font-weight: 100;
+      margin: 2em;
+    }
+  }
+  >.contributors {
+    >h3 {
+      font-weight: normal;
+      color: #565656;
+    }
+    >.contributors-list {
+      display: flex;
+      justify-content: space-evenly;
+      .contributor {
+        display: flex;
+        flex-direction: column;
+        > a {
+          text-decoration: none;
+          >.user-image {
+            border-radius: 0.25em;
+            margin: 0;
+          }
+          >.user-names {
+            margin: 0;
+            border: 2px gray solid;
+            border-radius: 0.25em;
+            text-align: left;
+            >.user-full-name {
+              color: black;
+              margin: 1em;
+            }
+            >.user-name {
+              color: gray;
+              margin: 1em;
+            }
+          }
+        } 
+      }
+    }    
+  }
+}
```

Comment:
It's driving me crazy. Not matter how many times I go over the files, I still keep missing those. :man_facepalming:

Reply:
Fixed

Reply:
You can fix it after creating a PR. It's easy, because all the `no new line` is highlighted.
Or you can find word `newline` in `git show`, before PR.
Anyway, we're working on app, which will fix all automatically.